libs/api/metadata-converter/src/lib/xml-utils.ts

```diff
@@ -312,6 +352,7 @@ export function appendChildren(
 ): ChainableFunction<XmlElement, XmlElement> {
   return (element) => {
     if (!element) return null
+    childrenFns = childrenFns.filter((fn) => fn)
```

Comment:
yes that makes sense! Maybe documenting this behavior (i.e. not appending a child if it is falsy) would be good?

Reply:
Do you mean in the code, or did you have somewhere specifically in mind?

Reply:
in the code :)